docs/data-sources/ga_endpoint_groups.md

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "GA (Global Accelerator)"
```

Comment:
The subcategory in resource is `subcategory: "Global Accelerator (GA)"`.
Is there any special reason for writing it this way?

Reply:
done